redash/query_runner/athena.py

```diff
@@ -238,6 +249,9 @@ def run_query(self, query, user):
                     'athena_query_id': athena_query_id
                 }
             }
+            if EXPOSE_COST:
+                price = self.configuration.get('cost_per_tb', 5)
+                data['metadata'].update({'query_cost': '${0:.2f}'.format(price * qbytes * 10e-12)})
```

Comment:
Let's store it as a number and format it in view, in case someone would like to run calculations on it.